executor/show.go

```diff
@@ -1677,6 +1678,28 @@ func (e *ShowExec) fetchShowCreateUser(ctx context.Context) error {
 		tokenIssuer = " token_issuer " + tokenIssuer
 	}
 
+	maxQuestions := rows[0].GetUint64(10)
+	maxUpdates := rows[0].GetUint64(11)
+	maxConnections := rows[0].GetUint64(12)
+	maxUserConnections := rows[0].GetUint64(13)
+	var resourceStr string
+	if maxQuestions > 0 || maxUpdates > 0 || maxConnections > 0 || maxUserConnections > 0 {
```

Comment:
We need to add some test cases to cover this code snippet.

Reply:
Done in 8b8ba8b648f24fabbd4c0e2fcf9d6e5bdde97a4b